README.md

```diff
@@ -194,6 +194,9 @@ let g:nv_create_note_window = 'vertical split'
 " Boolean. Show preview. Set by default. Pressing Alt-p in FZF will toggle this for the current search.
 let g:nv_show_preview = 1
 
+" Boolean. Respect .*ignore files in or above nv_search_paths. Set by default.
+let g:nv_ignore_files = 1
```

Comment:
Rename to something like `g:nv_use_ignore_files` and update the `s:` var too. `nv_ignore_files` feels vague to me.